subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/ParallelStaleOutputIntegrationTest.groovy

```diff
@@ -81,22 +79,11 @@ class ParallelStaleOutputIntegrationTest extends AbstractIntegrationSpec {
         """
 
         expect:
-        fails("a:foo", "b:foo", "--parallel")
+        executer.expectDocumentedDeprecationWarning("Configuring project ':a' without an existing directory is deprecated. The configured projectDirectory '${testDirectory.file('a').path}' does not exist, can't be written to or is not a directory. This behavior has been deprecated. This will fail with an error in Gradle 9.0. Make sure the project directory exists and can be written. Consult the upgrading guide for further information: https://docs.gradle.org/current/userguide/upgrading_version_8.html#deprecated_missing_project_directory")
```

Comment:
Is it simple to fix these other unrelated deprecation warnings?

Reply:
It was.